#CREATE request parameters

string jointname
--------------------------------
#CREATE response parameters
float64[] ef